docs/blog/_posts/2020-12-18-scala3-m3.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: blog-page
+title: Scala 3.0.0-M3: developer's preview before RC1
+author: Anatolii Kmetiuk
+authorImg: /images/anatolii.png
+date: 2020-12-18
+---
+We are happy to announce the release of Scala 3.0.0-M3. This release is the Developer's Preview release intended to contain all the features meant for RC1 which is tentatively planned for January 2021. The purpose of M3 is to give the community a chance to try out all the features and give us feedback before sealing them in RC1.
+
+To collect the feedback, we have designed a [Scala 3 Developer’s Preview satisfaction survey (3-5 min)](https://docs.google.com/forms/d/e/1FAIpQLSflVmTu9lhrtnSTh2tKAjUGrt3WvEgwlDqZg66O3EVSXd1aJg/viewform?usp=sf_link) – please fill it in to express how you feel about Scala 3.
+
+For the weeks between M2 and M3, we were tying up all the loose ends so that to end 2020 having done everything intended for the release. Also, we have used this time as the last chance to review and fine-tune the language syntax. A large number of consultations with the community took place during this time so that to ensure everyone is on the same page.
+
+In this article, you will find the summary of the most important changes that took place.
+
```

Comment:
Please, once this blog is merged: https://github.com/scala/scala-lang/pull/1195 add something like:

To read more about all things that surround the Scala 3 release in the next months, check out the "Scala 3 - Crossing the finish line" blog post on scala-lang.org